src/java.base/share/classes/java/nio/file/WatchEvent.java

```diff
@@ -48,6 +48,7 @@
 
     /**
      * An event kind, for the purposes of identification.
+     * @param <T> the type of the context value
```

Comment:
This is okay but the it differs slightly to the type parameters specified further up. I think the issue here is that it was just wasn't copied down to WatchEvent.Kind.

Reply:
Okay -- I'll for the T type parameter of the Kind interface I'll reuse the wording of the T type parameter of the enclosing WatchEvent interface. (The type variable on Kind could be renamed to show that the two type parameters are distinct.)